dspace/config/spring/api/discovery.xml

```diff
@@ -61,6 +61,11 @@
                 <entry key="publicationVolume" value-ref="publicationVolume"/>
                 <entry key="periodical" value-ref="periodical"/>
                 <entry key="project" value-ref="project"/>
+                <!-- OpenAIRE4 guidelines - search for a creator that can be a Person
+                     or an OrgUnit -->
+                <entry key="openAIRECreator" value-ref="personOrOrganization"/>
```

Comment:
Tiny thing, but this `key` could be renamed to simply be `personOrOrganization` as well. This is no longer OpenAIRE specific.  The comment also could be updated to be more generic, for example: `Search across Person and Organization entities (used for OpenAIRE4)`

Reply:
Done.